Neos.ContentRepository.Export/Tests/Behavior/Features/Bootstrap/CrImportExportTrait.php

```diff
@@ -167,6 +144,66 @@ public function iExpectTheFollowingJsonL(PyStringNode $string): void
         Assert::assertSame($string->getRaw(), ExportedEvents::fromIterable($eventsWithoutRandomIds)->toJsonl());
     }
 
+    /**
+     * @Then I expect the following events to be exported
+     */
+    public function iExpectTheFollowingEventsToBeExported(TableNode $table): void
+    {
+
+        if (!$this->crImportExportTrait_filesystem->has('events.jsonl')) {
+            Assert::fail('No events were exported');
+        }
+        $eventsJson = $this->crImportExportTrait_filesystem->read('events.jsonl');
+        $exportedEvents = iterator_to_array(ExportedEvents::fromJsonl($eventsJson));
+
+        $expectedEvents = $table->getHash();
+        foreach ($exportedEvents as $exportedEvent) {
+            $expectedEventRow = array_shift($expectedEvents);
+            if ($expectedEventRow === null) {
+                Assert::assertCount(count($table->getHash()), $exportedEvents, 'Expected number of events does not match actual number');
+            }
+            if (!empty($expectedEventRow['Type'])) {
+                Assert::assertSame($expectedEventRow['Type'], $exportedEvent->type, 'Event: ' . $exportedEvent->toJson());
+            }
+            try {
+                $expectedEventPayload = json_decode($expectedEventRow['Payload'], true, 512, JSON_THROW_ON_ERROR);
+            } catch (\JsonException $e) {
+                throw new \RuntimeException(sprintf('Failed to decode expected JSON: %s', $expectedEventRow['Payload']), 1655811083);
+            }
+            $actualEventPayload = $exportedEvent->payload;
+            foreach (array_keys($actualEventPayload) as $key) {
+                if (!array_key_exists($key, $expectedEventPayload)) {
+                    unset($actualEventPayload[$key]);
+                }
+            }
+            Assert::assertEquals($expectedEventPayload, $actualEventPayload, 'Actual event: ' . $exportedEvent->toJson());
+        }
+        Assert::assertCount(count($table->getHash()), $exportedEvents, 'Expected number of events does not match actual number');
+    }
+
+    /**
+     * @Then I expect the following sites to be exported
+     */
+    public function iExpectTheFollowingSitesToBeExported(TableNode $table): void
```

Comment:
we should also test this `I expect the following sites to be exported` for Neos.Neos (not only the legacy migrations probably) ... and move this method to Neos.Neos then?